app/src/main/res/layout/simple_options_recycler.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:paddingTop="20dp">
```

Comment:
I personally prefer to have the padding in the `Navigation bar order` settings dialog to be honest.